egregoria/src/vehicles/data.rs

```diff
@@ -14,12 +14,16 @@ use serde::{Deserialize, Serialize};
 use specs::{Builder, Entity, World, WorldExt};
 use specs::{Component, DenseVecStorage};
 
-pub const TIME_TO_PARK: f32 = 5.0;
+// How close a vehicle should be to the start of its itinerary before it decides it's good enough and starts driving.
+pub const DISTANCE_FOR_UNPARKING: f32 = 5.0;
+
+// The duration for the parking animation.
+pub const TIME_TO_PARK: f32 = 4.0;
 
 #[derive(Copy, Clone, Debug, Serialize, Deserialize)]
 pub enum VehicleState {
     Parked(ParkingSpotID),
-    ParkedToRoad(Spline, f32),
+    ParkedToRoad(Spline),
```

Comment:
I don't think the spline is needed anymore since the points end up in local_path